packages/protocol/contracts/dapis/RrpDapiServer.sol

```diff
@@ -0,0 +1,178 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.4;
+
+import "../AirnodeRrp.sol";
+import "./reducers/MeanMedianHybrid.sol";
+
+contract RrpDapiServer is MeanMedianHybrid {
+    struct Dapi {
+        uint256 noResponsesToReduce;
```

Comment:
dAPI reduces responses only once, when there are this many responses

Reply:
nit: when I first read this variable name, i thought it said "no responses to reduce" and that it was a boolean. why not opt for something like `requiredMin`

Reply:
Yeah I know you don't like no- as in "number of". I actually scratched my head a bit looking for an alternative. `requiredMin` is definitely worse. I think we can keep it as placeholder and replace it when we come up with a better name.